microshift_release_notes/microshift-4-17-release-notes.adoc

```diff
@@ -68,9 +68,14 @@ This release adds improvements related to the following components and concepts.
 //[id="microshift-4-17-configure-route-admission-policy_{context}"]
 //==== Configuring the route admission policy now available
 
-[id="microshift-4-17-storage"]
+[id="microshift-4-17-storage_{context}"]
 === Storage
 
+[id="microshift-4-17-LVMS-resource-footprint-reduction_{context}"]
+==== Support for LVM storage resource footprint reduction
+With this release, LVM storage resource footprint reduction is supported in {microshift-short}. For more information, see
```

Comment:
I dont understand how resource reduction can be supported. +1 to @DanielFroehlich suggestions

Reply:
changes are done